src/ls.js

```diff
@@ -128,3 +149,16 @@ function _ls(options, paths) {
   return list;
 }
 module.exports = _ls;
+
+
+function ls_stat(path) {
+  var stats = fs.statSync(path);
+  // Note: this object will contain more information than .toString() returns
+  var long_stats = Object.create(stats);
+  long_stats.name = path;
```

Comment:
I'd suggest making this value just the filename instead of the entire path, and then adding another property called `path` with the full path. Seems a little clearer that way IMO.